package.json

```diff
@@ -16,18 +16,19 @@
     "url": "https://github.com/Rufasa85/callsign/issues"
   },
   "dependencies": {
-    "mongoose": "~4.3.1",
+    "bcrypt": "~0.8.5",
+    "body-parser": "~1.14.1",
     "browserify": "^12.0.1",
     "express": "^4.13.3",
+    "express-jwt": "~3.3.0",
     "gulp": "^3.9.0",
+    "jsonwebtoken": "~5.4.1",
+    "lodash": "^3.10.1",
```

Comment:
This is a weird diff because when I installed lodash, it alphabetized everything.